### ISSUE TITLE
Integration tests for Cloud Sql connection via unix domain socket

### ISSUE THREAD

**Issue by mpeddada1** [GoogleCloudPlatform/cloud-sql-jdbc-socket-factory#1032]
Before we can relocate [these](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/88a02eabe6c3b248e12bb485ac5785ad612086d4/docs/jdbc-mysql.md#connection-via-unix-sockets)  native image configurations to the repo, we will need some tests that can be rn against native image compilation to verify compatibility. 

cc/ @suztomo, @shubha-rajan

**Comment by enocom** [comment]
Closing this as a duplicate of https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/issues/217. We'll track all GraalVM work there.
